server/proxy/bindings.go

```diff
@@ -194,7 +194,9 @@ func (p *Proxy) scanAppBindings(app *apps.App, bindings []*apps.Binding, locPref
 			b.Bindings = scanned
 		}
 
-		p.cleanForm(b.Form)
+		if b.Form != nil {
+			cleanForm(*b.Form)
```

Comment:
Since `cleanForm` is returning a form now, and not mutating a pointer, we need to reassign here, and possibly doing something with the errors returned

```suggestion
                var problems []error
		if b.Form != nil {
			b.Form, problems = cleanForm(*b.Form)
```

Reply:
@mickmister: done, please check